addon/services/gwr.js

```diff
@@ -81,16 +81,17 @@ export default class GwrService extends Service {
       .buildXMLRequest(xmlMethod, query)
       .replace(/\r?\n|\r/g, "");
 
-    response = await this.authFetch.fetch(
-      query.page
-        ? `/${urlPath}?page=${query.page}&size=${this.pageSize}`
-        : `/${urlPath}`,
-      {
-        headers: {
-          query: queryXML,
-        },
-      }
-    );
+    let fetchQuery = query.page
+      ? `/${urlPath}?page=${query.page}&size=${this.pageSize}`
+      : `/${urlPath}`;
+    if (query.sortColumn) {
+      fetchQuery = `${fetchQuery}&sortColumn=${query.sortColumn}&sortDirection=${query.sortDirection}`;
+    }
+    response = await this.authFetch.fetch(fetchQuery, {
+      headers: {
+        query: queryXML,
+      },
+    });
```

Comment:
Maybe use URLSearchParam for better readability

Reply:
Done